client/src/components/EditProfile/AvailabilityTab/AvailabilityRow/useStyles.ts

```diff
@@ -0,0 +1,27 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  availabilityRow: {
+    display: 'grid',
+    gridTemplateColumns: '30fr 35fr 35fr',
```

Comment:
You can assign the styles above by using MUI `Grid` containers/items above rather than `Box` components